src/__Private/LintRunConfig.hack

```diff
@@ -39,21 +51,10 @@ final class LintRunConfig {
     // Defaults to false.
     ?'disableAllAutoFixes' => bool,
     // Override any of the above for a subset of files in the project
-    ?'overrides' => vec<
-      shape(
-        // Which files this override applies to (uses `fnmatch()`)
-        'patterns' => vec<string>,
-        ?'builtinLinters' => NamedLinterGroup,
-        ?'extraLinters' => vec<string>,
-        ?'disabledLinters' => vec<string>,
-        ?'disabledAutoFixes' => vec<string>,
-        ?'disableAllAutoFixes' => bool,
-        ?'disableAllLinters' => bool,
-      )
-    >,
+    ?'overrides' => vec<self::TOverride>,
     // Each linter may specify a type for itself.
     // The type of this key is effectively `dict<classname<T1 ... Tn>, Tx>`
-    ?'linterConfigs' => dict<string, dynamic>,
+    ?'linterConfigs' => dict<string, shape(...)>,
```

Comment:
revert to dynamic, and use dynamic for the new option

https://github.com/hhvm/hhast/pull/364#discussion_r707531021